migrations/0008_add_server-settings.sql

```diff
@@ -0,0 +1,11 @@
+-- Migration number: 0003   2023-02-24T15:03:27.478Z
+
+CREATE TABLE IF NOT EXISTS server_settings (
+  setting_name TEXT UNIQUE NOT NULL,
+  setting_value TEXT
+);
+
+CREATE TABLE IF NOT EXISTS server_rules (
+  id INTEGER PRIMARY KEY,
```

Comment:
```suggestion
  id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Reply:
the value is autoincrementing even without that though 😮